INSTALL.md

```diff
@@ -26,33 +25,41 @@ source code.  Unpack as follows.
     tar xzf <some dir>/accumulo-X.Y.Z-bin.tar.gz
     cd accumulo-X.Y.Z
 
+There are four scripts in the the `bin/` directory that are use to manage Accumulo:
+
+1. `accumulo` - Runs Accumulo command-line tools and starts Accumulo processes
+2. `accumulo-service` - Runs Accumulo processes as services
+3. `accumulo-single`- Manages Accumulo on a single node
+4. `accumulo-cluster` - Manages Accumulo on a cluster
```

Comment:
The `accumulo-cluster` command requires the host files (tservers, master, gc, etc) to be created.  I created the the `accumulo-single` to avoid having to create them.  However, I think you are right about removing it. It's pretty easy to create them using `accumulo-cluster bootstrap-config` and it would be nice to reduce the number of scripts in bin/.